resources/views/bread/partials/form-builder.blade.php

```diff
@@ -0,0 +1,116 @@
+            @foreach($htmlRow->panels as $key_panel => $panel)
+                <div class="{{ isset($panel->class) ? $panel->class : '' }}">
+                    <div class="panel-heading">
+                        <h3 class="panel-title"><i class="icon wb-image"></i> @isset($panel->title) {{ __($panel->title) }} @endisset</h3>
+                        <div class="panel-actions">
+                            <a class="panel-action voyager-angle-down" data-toggle="panel-collapse" aria-hidden="true"></a>
+                        </div>
+                    </div>
+                    <div class="panel-body">
+                        @php
+                            $formBuilderFields->push($panel->fields);
+                        @endphp
+                        @foreach($dataTypeRows->whereIn('field', $panel->fields) as $row)
+                            <!-- GET THE DISPLAY OPTIONS -->
+                            @php
+                                $options = json_decode($row->details);
+                                $display_options = isset($options->display) ? $options->display : NULL;
+                            @endphp
+
+                            @if ($options && isset($options->formfields_custom))
+                                @include('voyager::formfields.custom.' . $options->formfields_custom)
+                            @else
+                                <div class="form-group @if($row->type == 'hidden') hidden @endif @if(isset($display_options->width)){{ 'col-md-' . $display_options->width }}@else{{ 'col-md-12' }}@endif" @if(isset($display_options->id)){{ "id=$display_options->id" }}@endif>
+                                    {{ $row->slugify }}
+                                    <label for="name">{{ $row->display_name }}</label>
+                                    @include('voyager::multilingual.input-hidden-bread-edit-add')
+                                    {!! app('voyager')->formField($row, $dataType, $dataTypeContent) !!}
+
+                                    @foreach (app('voyager')->afterFormFields($row, $dataType, $dataTypeContent) as $after)
+                                        {!! $after->handle($row, $dataType, $dataTypeContent) !!}
+                                    @endforeach
+                                </div>
+                            @endif
+                        @endforeach
+                    </div><!-- panel-body -->
+
+                    @if(!$key && count($htmlRow->panels) == $key_panel+1)
+                    <div class="panel-footer">
+                        <button type="submit" class="btn btn-primary save">{{ __('voyager.generic.save') }}</button>
+                    </div>
+                    @endif
+                </div>
+            @endforeach
+        </div>
+    @endforeach
+
+    @php
+        $formBuilderFields = $formBuilderFields->collapse()->toArray();
+        $notFormBuilderFields = $dataTypeRows->whereNotIn('field', $formBuilderFields)
```

Comment:
This line is breaking the TravisCI tests.  `whereNotIn()` is not valid in Laravel 5.3, which we still support.